crates/nargo/src/lib.rs

```diff
@@ -1,4 +1,9 @@
 #![forbid(unsafe_code)]
+#![warn(unused_crate_dependencies, unused_extern_crates)]
+
+// Necessary as we use `color_eyre` in `main.rs`.
```

Comment:
Yeah the docs said that this is a false positive that pops up

Reply:
Yep, we can also avoid this with https://github.com/noir-lang/noir/pull/907

Reply:
Ah okay, if we can get 907 merged today, then it would be good to remove this in a later PR